auth_signup_verify_email/controllers/main.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from openerp import _, http
+from openerp.addons.auth_signup.controllers.main import AuthSignupHome
+
+
+_logger = logging.getLogger(__name__)
+
+
+class SignupVerifyEmail(AuthSignupHome):
+    @http.route()
+    def web_auth_signup(self, *args, **kw):
+        if (http.request.params.get("login") and
+                not http.request.params.get("password")):
+            return self.passwordless_signup(http.request.params)
+        else:
+            return super(SignupVerifyEmail, self).web_auth_signup(*args, **kw)
+
+    def passwordless_signup(self, values):
+        qcontext = self.get_auth_signup_qcontext()
+
+        # Check good format of e-mail
+        from validate_email import validate_email
```

Comment:
Could you check next guideline https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror please?